criteria/version-control-and-history.md

```diff
@@ -64,3 +67,5 @@ For example, adding a new category of applicant to a codebase that manages grant
 * [Producing OSS: Version Control Vocabulary](https://producingoss.com/en/vc.html#vc-vocabulary) by Karl Fogel.
 * [Maintaining version control in coding](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding) by the UK Government Digital Service.
 * [Semantic Versioning Specification](https://semver.org/) used by many codebases to label versions.
+* [GitHub Learning Lab](https://lab.github.com/) for learning git or refresh your skills.
```

Comment:
The GitHub learning lab teaches mostly how to use GitHub and not a lot of Git. For learning Git in depth I'd recommend [ProGit](https://git-scm.com/book/en/v2) on the Git website.

```suggestion
* [GitHub Learning Lab](https://lab.github.com/) for learning how to use GitHub or refresh your skills.
```